### ISSUE TITLE
Release note generator should add contributors list

### ISSUE THREAD

**Issue by katre** [bazelbuild/bazel#6544]
Currently, we're manually adding the list of contributors to thank to the release notes: we should instead be generating this.

Current script:
```
git log <previous version baseline>..<current version baseline> --format="%aN <%aE>" | sort | uniq | grep -v 'google\.com' | cut -d '<' -f 1
```

**Comment by jin** [comment]
A small enhancement: discovering the issue(s) the contributor's commit closes, and linking to those issues too.

e.g.

```
This release contains contributions from many people at Google, as well as:

* Jane Doe: #1234 Fixes foo bug, #1235 Added bar feature
* John Doe: #1236 Improved baz performance
```

**Comment by katre** [comment]
@jin Currently we're not linking individual contributors to single issues, just listing everyone who contributed. I'm not sure trying to do that is going to be useful in the release notes: anyone who is interested can check the Git history themselves.
